lib/smart_answer_flows/coronavirus-find-support.rb

```diff
@@ -0,0 +1,277 @@
+module SmartAnswer
+  class CoronavirusFindSupportFlow < Flow
+    def define
+      name "coronavirus-find-support"
+      start_page_content_id "6a6ab3c9-4612-4764-8f2f-2c534c3c6b19"
+      flow_content_id "31d81949-aa15-48cf-a7f1-f0d0a670e8db"
+      status :draft
+
+      # ======================================================================
+      # What do you need help with because of coronavirus?
+      # ======================================================================
+      checkbox_question :need_help_with? do
+        option :feeling_unsafe
+        option :paying_bills
+        option :getting_food
+        option :being_unemployed
+        option :going_to_work
+        option :somewhere_to_live
+        option :mental_health
+        option :not_sure
+
+        on_response do |response|
+          self.calculator = Calculators::CoronavirusFindSupportCalculator.new
+          calculator.need_help_with = response
+        end
+
+        next_node do
+          question :feel_safe?
+        end
```

Comment:
Thanks @theseanything - yes, something I missed (DOH!) - am in the process of fixing.

Reply:
Would there be a way to create a function to calculate the next node from `current question` and `needs_help_with` values? Might help remove the repetition in the `next_node` blocks - which I could see being a pain to refactor if the sections changed. Also would allow us to write unit tests - rather than relying on integration tests to check every path.

Reply:
Umm!  Thanks @theseanything should be possible - I'll take a look.  Thanks for the suggestion 👍

Reply:
Hi @theseanything - thanks for the suggestion, I prefer the new approach 👍   

However, there is one slight issue - because (like `benefit-cap-calculator`) the smart answer is using a method to obtain the next question in the `next_node` block, I have had to add it to the `GraphPresenter` `EXEMPTIONS_LIST`.  The flow does still actually render (again like the `benefit-cap-calculator`) albeit it not quite correctly.  I feel this is a bug with the `Flow` not being able to run or follow the path correctly.  Something that probably needs addressing, but not as part of this PR.